### PR TITLE
HTML Importer

### PR DESCRIPTION
**Pull request by ericaxu**
Adds `HTMLImporter` that import files the naive way. Builds on the `refactor` branch.

### CODE REVIEW

File: html.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                let folder = app.vault.getAbstractFileByPath(outputFolder);
+
+                if (folder === null || !(folder instanceof TFolder)) {
+                        await app.vault.createFolder(outputFolder);
+                        folder = app.vault.getAbstractFileByPath(outputFolder);
+                }
+
+                for (let path of filePaths) {
+                        let htmlContent = await fs.readFileSync(path, 'utf-8');
+                        let mdContent = htmlToMarkdown(htmlContent);
```
Comment (lishid) [inline html.ts:34]: We should move everything into the try-catch block.

File: html.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+                        try {
+                                await this.saveAsMarkdownFile(folder as TFolder, baseFileName(normalizePath(path)), mdContent);
```
Comment (lishid) [inline html.ts:37]: Can we pre-verify folder to be TFolder?

**Review by lishid — COMMENTED** [review]
Need to run the auto formatter please.

**Comment by ericaxu** [comment]
> Need to run the auto formatter please.

On `html.ts`? I ran in and didn't notice a difference.

**Pull request merged**